frontend/src/app/main/streams.cljs

```diff
@@ -120,6 +120,21 @@
                        (rx/map (constantly false))))
                  (rx/dedupe))]
         (rx/subscribe-with ob sub)
+        sub))
+
+(defonce keyboard-ctrl
+  (let [sub (rx/behavior-subject nil)
+        ob  (->> (rx/merge
+                  (->> st/stream
+                       (rx/filter keyboard-event?)
+                       (rx/map :ctrl))
```

Comment:
This doesn't work. Apparently when you release ctrl key, an on-key-up is fired, but the :ctrl flag is still set.